.merlin

```diff
@@ -1,3 +1,3 @@
-PKG utop merlin_extend compiler-libs easy-format MenhirLib
+PKG utop merlin_extend compiler-libs easy-format MenhirLib str cmdliner
```

Comment:
Are we re-introducing `str` here? 

Related: https://github.com/facebook/reason/pull/659